starter-code/js/index.js

```diff
@@ -1,11 +1,55 @@
-var $cart = document.querySelector('#cart tbody');
-var $calc = document.getElementById('calc');
+var $cart = document.querySelector("#cart tbody");
+var $calc = document.getElementById("calc");
 
 function updateSubtot($product) {
   // Iteration 1.1
+
+  let name = $product.querySelector(".product>.name>span").innerHTML;
+  let price = parseFloat($product.querySelector(".product>.pu>span").innerHTML);
+  let quantity = parseFloat($product.querySelector(".quantity").value);
+  $product.querySelector(".subtot>span").innerHTML = price * quantity;
+  return price * quantity;
 }
 
 function calcAll() {
   // Iteration 1.2
+  let total = 0;
+  $cart.querySelectorAll(".product").forEach(function(product) {
+    total += updateSubtot(product);
+  });
+  document.querySelector("h2>span").innerHTML = total;
 }
-$calc.onclick = calcAll;
\ No newline at end of file
+$calc.onclick = calcAll;
+
+$cart.querySelectorAll(".product>.rm>.btn-delete").forEach(function(button) {
+  button.onclick = function(event) {
```

Comment:
this works but only for elements that are not created by the user. You still need to call this as well when you create the new product